### PR TITLE
add ivis parameters to avoid NoneType error

### PR DESCRIPTION
**Pull request by idroz**

### CODE REVIEW

**Comment by duhaime** [comment]
@idroz thanks very much for this! I confess I tinkered with ivis a bit and was excited to integrate it into this project but all of my models have been degenerates and have encoded all inputs to the same location--do you have a hunch why that might be?

Is it necessary to train a custom model for one's dataset, or will any of the pretrained models generalize to a wide range of high-dimensional point datasets?

**Comment by idroz** [comment]
@duhaime I think the issue may be around how the data is scaled. ivis uses a distance-based triplet loss function, which is sensitive to distances between observations. Scaling the data to unit variance and unit standard deviation should produce appropriate transformations.

ivis can be trained on a corpus once and pertained models will generalise (provided that your input sequence of words/sentences is padded consistently). One of the strength of the algorithm is flexibility in base networks architectures. So to create an "general" ivis model for an English language corpus, you could create a custom NN:

```python
def get_cnn_model(vocab_size, weights=None):
    embed_size = 50
    inp = Input(shape=(maxlen, ))
    x = Embedding(vocab_size, 50,
                  weights=[weights],
                  input_length=maxlen)(inp)
    x = Dropout(0.2)(x)
    x = Conv1D(250, 3, padding='valid', activation='relu', strides=1)(x) 
    x = GlobalAveragePooling1D()(x)
    x = Dense(250)(x)
    x = Dropout(0.2)(x)

    model = Model(inputs=inp, outputs=x)
    return model
```

and then pass it into ivis:

```python
ivis = Ivis(model=get_cnn_model(), n_epochs_without_progress=5)
ivis.fit(X)
```

Where X is your sequence of words/sentences.

**Comment by duhaime** [comment]
@idroz Thanks so much for this incredible follow up! I hadn't recognized that providing a custom model literally just passing a Keras model in--that's awesome!

I'm hoping to try out some custom models a little later this week--thanks again for this writeup!
